styles.css

```diff
@@ -0,0 +1,98 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  background-color: #e5e5e5;
+}
+
+.header {
+  height: 80px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  background-color: white;
+}
+
+/* Hero section */
+.section-hero {
+  background-image: url("./image/header-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  padding: 100px 0;
+  background-color: white;
+  border-bottom-left-radius: 8rem;
+}
```

Comment:
- [ ] Kindly align your background image correctly in order with the provided Figma design. :wink: 
    **_Hint:_** Add this  `background-size: cover` and see how it affects your design.

    - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/198536426-5390900f-7ba8-48d5-ae1e-a8dcacc80ad1.png)

   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/198536578-37f8ffe2-7568-4302-a120-ee958e2635da.png)